src/test/features/paid-in-full/routes/checks/authorization-check.ts

```diff
@@ -0,0 +1,5 @@
+import { checkAuthorizationGuards as check } from 'test/routes/authorization-check'
+
+export function checkAuthorizationGuards (app: any, method: string, pagePath: string) {
```

Comment:
do we really need this wrapper?

Reply:
If I remove the wrapper and change the 'check' call to checkAuthorizationGuards, I'm given an "Import conflicts with local declaration of checkAuthorizationGuards" error

Reply:
I mean you don't need this file at all. Use `checkAuthorizationGuards` directly where needed